src/properties/triangles.c

```diff
@@ -260,7 +262,7 @@ int igraph_transitivity_local_undirected2(const igraph_t *graph,
 
     neis = IGRAPH_CALLOC(no_of_nodes, long int);
     if (neis == 0) {
-        IGRAPH_ERROR("local transitivity calculation failed", IGRAPH_ENOMEM);
+        IGRAPH_ERROR("Insufficient memory for local transitivity calculation.", IGRAPH_ENOMEM);
```

Comment:
I would not change this; the default error handler shows an explanation of `IGRAPH_ENOMEM` by default and it already includes the fact that it is an out-of-memory condition.

Reply:
Okay, nevermind. I see a tendency in recent PRs where the error messages are designed to be self-explanatory now such that the error code is only an "auxiliary" bit of information but not an essential one. All this points in the direction that in the near future the higher level interfaces do not have to add `igraph_strerror(error_code)` at the end of the error message to ensure that the user gets the full picture. I'm okay with the current version of the error message in this PR.

Reply:
OK, I'll leave it then. I do not have strong feelings about this topic. I really just wanted to capitalize it and add a fullstop, but then I ended up changing the message too.